scripts/us_bls/jolts/bls_jolts.py

```diff
@@ -184,17 +226,35 @@ def row_to_stat_var(row):
         seasonal_adjustment = row['seasonal_adjustment']
 
         return (
-            f"dcs:{base_stat_var}_NAICS_{industry_code}_{seasonal_adjustment}")
+            f"dcs:{base_stat_var}_NAICS{industry_code}_{seasonal_adjustment}")
 
     # Build map to Statistical Variable.
     jolts_df['seasonal_adjustment'] = jolts_df['seasonal'].apply(
         lambda adj: "Adjusted" if adj == "S" else "Unadjusted")
     jolts_df['StatisticalVariable'] = jolts_df.apply(row_to_stat_var, axis=1)
+    for old, new in _dcid_map.items():
+        jolts_df['StatisticalVariable'] = jolts_df[
+            'StatisticalVariable'].str.replace(old, new, regex=False)
     jolts_df['Value'] = jolts_df['value']
 
     return jolts_df, schema_mapping
 
 
+def remap_dcid():
+    file_path = 'BLSJolts_StatisticalVariables.mcf'
+    for line in fileinput.input(file_path, inplace=True):
+        line = line.rstrip()
+        if not line:
+            continue
+        for f_key, f_value in mcf_config._mcf_map.items():
+            if f_key in line:
+                if len(line) == len(f_key):
```

Comment:
instead of a loop to check against each entry, can't we lookup the entry by key:
  ```
  f_value  = mcf_config._mcf_map.get(line):
  if f_value:
    line = f_value
```